### PR TITLE
Homepage update

### PR DESCRIPTION
**Pull request by stockholmux**
### Description

- Adds feature repos to side bar
- Fixes misc issues with formatting and duplication
- Adds temporary lorem ipsum for content measurement (to be replace by subsequent PR)

Screenshot 2024-04-11 at 8 44 08 PM

### Issues Resolved

n/a

### Check List
- [x] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by madolson — DISMISSED** [review]
Sure, I'm fine with placeholder text for the moment.

File: index.markdown [hunk 1]
```diff
@@ -32,25 +32,19 @@ sidebar:
...
-Sed tempus sed lorem in blandit.
-
-Nulla vel lacus non elit facilisis commodo. Pellentesque venenatis eleifend orci.
-Maecenas ac turpis placerat, congue nisl ac, egestas erat.
-Donec aliquam congue mollis.
-Ahasellus a felis mi.
-Nullam volutpat ante purus, in aliquet arcu vestibulum eu.
+Valkey provides your applications with rich yet uncomplicated, high-performance data storage. Valkey is operationally flexible: you can run it as a **standalone** daemon or in a **cluster**, with options for **replication** and **high availability**. It was built from the ground up to operate **in-memory** to **reduce latency** and **increase throughput** for both reads and writes.
+
+In Valkey, data is accessed with keys and you model your information into **data structures**. You can operate on data structures in-place with a expressive collection of commands: no need for extra round trips or client process. Valkey has support for **strings**, **numbers**, **hashes**, **lists**, **sets**, **sorted sets**, **bitmaps**, hyperloglogs** and more.
```
Comment (madolson) [inline index.markdown]:
```suggestion
In Valkey, data is accessed with keys and you model your information into **data structures**. You can operate on data structures in-place with a expressive collection of commands: no need for extra round trips or client process. Valkey has support for **strings**, **numbers**, **hashes**, **lists**, **sets**, **sorted sets**, **streams**, hyperloglogs** and more.
```
Streams are more important than bitmaps imo.

File: index.markdown [hunk 2]
```diff
@@ -7,21 +7,44 @@ body_class: homepage
...
         links:
             -
                 title: 'Linux Foundation Launches Open Source Valkey Community'
                 url: https://www.linuxfoundation.org/press/linux-foundation-launches-open-source-valkey-community
 
----
\ No newline at end of file
+---
+
+Valkey provides your applications with rich yet uncomplicated, high-performance data storage. Valkey is operationally flexible: you can run it as a **standalone** daemon or in a **cluster**, with options for **replication** and **high availability**. It was built from the ground up to operate **in-memory** to **reduce latency** and **increase throughput** for both reads and writes.
```
Comment (madolson) [inline index.markdown:35]:
```suggestion
Valkey is an open source (BSD) high-performance key/value datastore that supports a variety workloads such as **caching**, **message queues**, and can act as a **primary database**.
Valkey can run as either a **standalone** daemon or in a **cluster**, with options for **replication** and **high availability**.
```

File: index.markdown [hunk 3]
```diff
@@ -7,21 +7,44 @@ body_class: homepage
...
+
+In Valkey, data is accessed with keys and you model your information into **data structures**. You can operate on data structures in-place with a expressive collection of commands: no need for extra round trips or client process. Valkey has support for **strings**, **numbers**, **hashes**, **lists**, **sets**, **sorted sets**, **bitmaps**, **hyperloglogs** and more.
```
Comment (madolson) [inline index.markdown:37]:
```suggestion
You operate on data structures in-place with a expressive collection of commands: no need for extra round trips or client process. Valkey has support for **strings**, **numbers**, **hashes**, **lists**, **sets**, **sorted sets**, **bitmaps**, **hyperloglogs** and more.
Valkey also supports native extensibility with built-in scripting support for **LUA** and supports plugins to create new commands, data types, and more.
```

Cut a sentence, and also suggest attaching the extensibility here.

File: index.markdown [hunk 4]
```diff
@@ -7,21 +7,44 @@ body_class: homepage
...
+
+You can also use Valkey to structure and distribute messages with the fire-and-forget simplicity of **built-in pub/sub messaging**. Valkey also has **streams**, a log-like structure that allows you to durably process information with multiple producers and across many consumers with a poll-free, zero wait mechanism. 
+
+You can script Valkey with a **built-in Lua** interpreter to optimize and finely control complex operations. 
+It also has a built-in **module extensibility API** that enables you to create new data types and commands that can exploit the full power of the storage engine and protocol.
+
+Valkey commands operate atomically and you can ensure atomicity across multiple keys and commands with **lock-free transactions**.
+
+You can use Valkey as a **database**, **cache**, **message broker**, and **message queue** as well as a storage engine for **session management**, **leaderboards**, **rate limiters**, and **real-time analytics workloads**.
```
Comment (madolson) [inline index.markdown:46]: Everything here has been merged earlier in my suggestion, so suggest cutting.
```suggestion
```

**Review by madolson — COMMENTED** [review]
Mostly suggesting chopping the main page down, it feels long.

**Comment by stockholmux** [comment]
@madolson Please re-review and I'll merge

**Review by madolson — APPROVED** [review]
LGTM for now. Thanks!

**Pull request merged**